### DISCUSSION TITLE
Open Source Contribution

### DISCUSSION THREAD

**Discussion by benjaminbowles** [CompositeLight/currentRMS-helper#49] [General]
Would you be up for contribution to the project from my developers?

**Comment by TechJed** [comment]
We contributed some code which got added (with some modifications)

**Comment by CompositeLight** [comment]
Absolutely up for that. If they're pros they might not be too impressed with my code(!), but that aside anything we can do to make Current work how it should works for me.

I've not really delved into understanding github beyond my own limited work-flow. From my basic understanding, the thing to do would be for you devs to fork the repo and then their additions / edits could be pushed back to the source. As far as I can see, it should be possible to fork it already, but I can't find a setting to enable/disable. Might need to create an organisation for it to exist under.

**Comment by CompositeLight** [comment]
Just been having a look at this, and forking does seem to be enabled as far as I can see. If anyone can't let me know. Also, if anyone wants to contribute in another way I'm totally open to that , just drop me a line. Thanks!
